src/core/objective.jl

```diff
@@ -277,24 +277,24 @@ function _objective_mc_min_fuel_cost_polynomial_linquad(pm::AbstractUnbalancedPo
             for (i,gen) in nw_ref[:gen]
                 bus = gen["gen_bus"]
 
-                #to avoid function calls inside of @NLconstraint:
+                #to avoid function calls inside of @constraint:
                 pg = var(pm, n, :pg, i)
                 pg = isa(pg, JuMP.Containers.DenseAxisArray) ? pg.data : pg
 
                 int_dim = length(pg)
                 if length(gen["cost"]) == 1
                     gen_cost[(n,i)] = gen["cost"][1]
                 elseif length(gen["cost"]) == 2
-                    gen_cost[(n,i)] = JuMP.@NLexpression(pm.model, gen["cost"][1]*sum(pg[i] for i in 1:int_dim) + gen["cost"][2])
+                    gen_cost[(n,i)] = JuMP.@expression(pm.model, (-gen["cost"][1]*sum(-pg[i] for i in 1:int_dim)) + gen["cost"][2])
```

Comment:
This should really not be needed. Let me take a look through the rest of the changes. Presumably there are now two signs that cancel each other out?

Reply:
It is very strange, we don't really understand why these are needed, but they fix a lot of the failing tests. The signs cancel out here, so this should be the exact same expression mathematically, but it affects the stability of the tests for some reason. Obviously, all of these tests pass with the old NLexpression syntax.